src/rtc/h5_prep.py

```diff
@@ -490,6 +490,13 @@ def get_metadata_dict(product_id: str,
     else:
         mosaic_snap_y = float(mosaic_snap_y)
 
+    # average azimuth pixel spacing
+    try:
+        average_zero_doppler_spacing_in_meters = \
+            burst_in.average_azimuth_pixel_spacing
+    except AttributeError:
+        average_zero_doppler_spacing_in_meters = '(UNSPECIFIED)'
```

Comment:
Hmm so why try/except? is it because you want to still allow to run with older s1reader versions?

Reply:
Hi @hfattahi , thank you for review! Yes. Although we'll run the latest S1-reader in production, I'd like to avoid breaking the code for people using previous versions of the S1-reader.